src/environmentd/tests/util.rs

```diff
@@ -346,6 +352,7 @@ pub fn start_server(config: Config) -> Result<Server, anyhow::Error> {
             postgres_factory,
             metrics_registry: metrics_registry.clone(),
             scratch_directory: None,
+            persist_pubsub_url: "http://localhost:6879".to_owned(),
```

Comment:
good spot. I chose to skip the pubsub server for sqllogictests since I don't think it'd get any useful coverage, but spun one up and passed in a real address for testdrive.

cc @def-